src/main/java/org/folio/circulation/item/controller/CirculationItemController.java

```diff
@@ -0,0 +1,46 @@
+package org.folio.circulation.item.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.circulation.item.service.CirculationItemsService;
+import org.folio.rs.domain.dto.CirculationItem;
+import org.folio.rs.rest.resource.CirculationItemIdApi;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static java.util.Objects.isNull;
+
+@Log4j2
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "/circulation-item/")
+public class CirculationItemController implements CirculationItemIdApi {
+
+  private final CirculationItemsService circulationItemsService;
+
+  @Override
+  public ResponseEntity<CirculationItem> createCirculationItem(String circulationItemId, CirculationItem circulationItem) {
+    log.info("createCirculationItem:: {}", circulationItem);
```

Comment:
improve this log , may be below
log.info("createCirculationItem:: creating CirculationItem with Id {}", circulationItemId);

Reply:
done